src/desktop/apps/collect/routes.coffee

```diff
@@ -7,8 +7,11 @@ page = new JSONPage name: 'browse-categories'
 GeocodedCities = require '../../collections/geocoded_cities'
 { COLLECT_PAGE_TITLES_URL } = require('sharify').data
 getCollectPageTitle = require '../../components/commercial_filter/page_title'
+splitTest = require '../../components/split_test/index.coffee'
 
 @index = (req, res, next) ->
+  # MERCH_SORT_TEST remove after test closes
+  splitTest('merch_sort_test').view()
```

Comment:
I wasn't entirely sure where this declaration should sit - I _think_ this triggers once per `/collect` view where it currently is.

Reply:
I would actually move this to be evaluated client-side, anywhere in the init function here: https://github.com/artsy/force/blob/76e1325fab1c153d66620481e47570193726f789/src/desktop/apps/collect/client.coffee should probably do it.

Since this fires a Segment call, client-side calls are automatically enhanced with a bunch of other information, whereas this is server-side (although that _might_ be sufficient for the AB test, but typically we do those client-side). That might actually be a good thing to add to the docs for `split_test`.

Reply:
Interesting, thanks - I'll move it to the client in that case.